rotkehlchen/db/filtering.py

```diff
@@ -92,6 +92,46 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
         return ['tx_hash=?'], [value]
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBReportsReportIDFilter(DBFilter):
```

Comment:
Okay I think that perhaps our guidance was bad. You misunderstood due to the comment to look at ethereum transactions. Ethereum transactions have a dedicated filter class only because they are really complicated and need to be queriable with a variety of combination of parameters.

Here all you need to do is filter by report id, offset and limit. Nothing more. There is no reason to get that complicated and add classes in the DB for filtering. Just filter directly in the dbhandler funciton